index.html

```diff
@@ -15,8 +15,10 @@ <h1>Guessing Game</h1>
         less?
       </p>
       <div>
-        <input type="number" id="guess" min="1" max="99" />
-        <button id="submit">Submit Guess</button>
+        <form id="guessNumberForm">
```

Comment:
Great! Thank you very much for your feedback. And yes, I thought it would be easier to activate the html elements' validation through the usage of a form. I am glad it was one good approach!